content/influxdb/v2.0/query-data/common-queries/count_unique_values_for_column.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Find and count unique values
+description: >
+  Count the number of unique values in a specified column.
+influxdb/v2.0/tags: [queries]
+menu:
+  influxdb_2_0:
+    name: Count unique values
+    parent: Common queries
+weight: 104
+---
+
+{{% note %}}
+This example uses [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data).
+{{% /note %}}
+
+The following examples identify and count unique locations that data was collected from.
+
+## Find unique values
+
+This example uses [`csv.from()`](influxdb/v2.0/reference/flux/stdlib/csv/from/), [`keep()`](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/keep/), and [`unique()`](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/selectors/unique/) to return unique values in a specified column.
+
+```js
+import "experimental/csv"
+
+csv.from(bucket: "noaa")
+  |> keep(columns: ["location"])
+  |> unique(column: "location")
+```
```

Comment:
- Since these are now just pulling from a bucket, you no longer need the "experimental/csv" import statment and `csv.from` becomes just `from`
- I think we should include `group()` in this example as well. Group will remove the default "series" grouping and return everything in a single table.